{{ cookiecutter.hosting_github_repo_name }}/web/public/index.html

```diff
@@ -1,43 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <!-- Primary Meta Tags -->
+    <title>
+      Council Data Project | {{ cookiecutter.municipality }} – Searchable city council videos, 
+      transcripts, voting records, and legislation.
+    </title>
+    <meta
+      name="title"
+      content="Council Data Project | {{ cookiecutter.municipality }} – Searchable city council videos, 
+      transcripts, voting records, and legislation."
+    />
+    <meta
+      name="description"
+      content="City council videos, transcripts, voting records, and legislation all searchable under 
+      a single website."
+    />
     <meta charset="utf-8" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
What will the favicon default to? You removed the favicon file.

Reply:
Oh weird, I thought I had replaced it with the favicon from CDP design. I'll go double-check that.

Reply:
Yeah, I double-checked on my end and there is indeed a favicon still 🧐.

Reply:
Force push maybe? If not then idk we will figure it out later..